safehttp/default_dispatcher.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package safehttp
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/google/safehtml"
+	"github.com/google/safehtml/template"
+	"net/http"
+)
+
+// DefaultDispatcher is responsible of writing the body of a response to an
+// Response Writer if it's of a safe type.
+type DefaultDispatcher struct{}
+
+// ContentType returns the Content-Type of a safe respons if it's of a safe-type
```

Comment:
`response` with an `e`

Reply:
FYI: @mattiasgrenfeldt check out the "add suggestion" option when making comments (the +- sign in the comment toolbar). This allows the PR author to apply the suggestions from GitHub's web UI with one click.